releases/patch-releases.md

```diff
@@ -1,117 +1,108 @@
+To discuss a cherry-pick, schedule, or anything else related to
+patch releases, contact the team as a whole or the appropriate member if
+there is a specific one:
 
-Cherry-picks deadline is the end of day by which all pending PRs to release-1.13
-branch have to merge to be included in the patch release. Any PRs left pending past
-this point will either wait for the next patch release, or may in rare cases
-delay current release.
+* Email: the whole team is reachable via email to
+  [kubernetes-patch-release-team].
+* Slack: You can just tag appropriate individuals or start a broader
+  conversation on #sig-release, as well as reach out to team members
+  via direct message.
 
-Target date is the date on which the patch release is cut. Must be at least 2
-days after cherry-picks deadline to allow all release-blocking tests to run.
+Please give us a business day to respond - we may be in a different timezone!
 
-| **Patch Release** | **Cherry-picks deadline** | **Target date** | **Owner** |
-| --- | --- | --- | --- |
-| 1.13.3 | TBA | 2019-01-24 | TBA |
-| 1.13.2 | 2019-01-08 | 2019-01-10 | @tpepper |
-| 1.13.1 | 2018-12-11 | 2018-12-13 | @aleksandra-malinowska |
+In between releases the team is looking at incoming cherry-pick
+requests on a weekly basis.  The team will get in touch with
+submitters via GitHub PR, SIG channels in Slack, and direct messages
+in Slack and email if there are questions on the PR.
 
-# 1.12
+## Cherry-Picks
 
-## Patch Release Management Team
+Please follow the [cherry-pick process].
 
-To discuss a cherry-pick, schedule, or anything else related to 1.12 patch releases,
-contact any member of the team. You can just tag us on #sig-release Slack channel
-or message directly. Please give us a business day to respond - we may be in a different timezone!
+Cherry-picks must be merge-ready in GitHub with proper labels (eg:
+approved, lgtm, release note) and passing CI tests ahead of the
+cherry-pick deadline.  This is typically two days before the target
+release, but may be more.  Earlier PR readiness is better, as we
+need time to get CI signal after merging your cherry-picks ahead
+of the actual release.
 
-Cherry-picks are monitored on regular basis, so it's likely you'll hear
-from us on PRs to release-1.12 branch by the time the next patch release is due.
+Cherry-pick PRs which miss merge criteria will be carried over and tracked
+for the next patch release.
 
-| **Github username** | **Slack username** |
-| --- | --- |
-| @feiskyer | @feisky |
+## Timelines
 
-## Timeline
+### 1.13
 
-Next patch release is **1.12.5**.
+Next patch release is **1.13.5**.
 
-Frequency of releases is likely to be about every 3 weeks. However, it can be affected
-by other factors, such as critical security fixes or holiday season.
+| **Patch Release** | **Cherry-picks deadline** | **Target date** | **Owner** |
+| --- | --- | --- | --- |
+| 1.13.5 | 25 March 2019 | 21 March 2019 | @aleksandra-malinkowska, @tpepper |
```

Comment:
nit: is there a need to deviate from ISO 8601 for the upcoming release date format?

Reply:
No reason, just a think-o on my part.  Amended.